website/components/PropTable.tsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import { css } from 'emotion';
+import { Table, TableHeader, Row, Cell } from '@leafygreen-ui/table';
+import { Subtitle, InlineCode } from '@leafygreen-ui/typography/dist';
+import PropDefinition from 'components/PropDefinition';
+import TypographyPropTable from 'components/TypographyPropTable';
+
+const subtitleBottomMargin = css`
+  margin-bottom: 24px;
+`;
+
+const tableBottomMargin = css`
+  margin-bottom: 56px;
+`;
+
+/**
+ * Syntactic units in unist syntax trees are called nodes.
+ * Interface dervied from Unified types
+ */
+export interface Node {
+  /**
+   * The variant of a node.
+   */
+  type: string;
+
+  /**
+   * Information from the ecosystem.
```

Comment:
what's an ecosystem?

Reply:
see above

Reply:
it would be informative to have more context around these comments. are there links to docs that can be included?